pkg/pillar/devicenetwork/dnc.go

```diff
@@ -651,6 +655,11 @@ func (ctx *DeviceNetworkContext) doUpdatePortConfigListAndPublish(
 	current := getCurrentDPC(ctx) // Used to determine if index needs to change
 	currentIndex := ctx.DevicePortConfigList.CurrentIndex
 	oldConfig, _ := lookupPortConfig(ctx, *portConfig)
+	if strings.Contains(portConfig.Key, "override") {
```

Comment:
Can you please put a // XXX remove this code when the controller supports WiFi config
comment so we can find it and zap it later?